src/isomorphic/children/__tests__/ReactChildren-test.js

```diff
@@ -87,6 +88,8 @@ describe('ReactChildren', function() {
     var mappedChildren = ReactChildren.map(instance.props.children, callback);
     expect(callback).toHaveBeenCalledWith(simpleKid, 0);
     expect(nthChild(mappedChildren, 0)).toBe(simpleKid);
+
+    expect(console.error.calls.length).toBe(1);
```

Comment:
IMO it's good practice to make sure that what's being logged is exactly what you expect - reading this without context, I have no idea why `console.error` is being logged to, which makes the tests harder to understand. If you check for the specific message that will help a lot with comprehension.